### PR TITLE
Add TESTING_GUIDELINES.md

### PR DESCRIPTION
**Pull request by dawiddylowicz**
Added testing guidelines to make test setup, test execution and test maintenance easier for developers.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-245-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://245-pr-onfido-sdk-ui-onfido.surge.sh

File: test/TESTING_GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+**Custom run:**
+
+In `test/` directory:
+1. Run `bundle install` to install ruby dependencies
+2. Run `bundle exec cucumber BROWSER=chrome SDK_URL=${SDK_URL} USE_SECRETS=false SEED_PATH=false DEBUG=false`, where you can customize:
+    - `BROWSER` - browser to run tests against (possible choices: `chrome`, `firefox`)
+    - `SDK_URL` - url to deployed SDK (possible choices: any surge link, or `"https://localhost:8080/?async=false"` when webpack started locally using `npm run dev`.
+    **Note:** It's very important to add `?async=false` GET parameter to the end of any URL so that tests wait for JWT to be generated)
+
+## Maintanance
```
Comment (hjheath) [inline test/TESTING_GUIDELINES.md]: spelling?!

File: test/TESTING_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Testing guidelines
+
+Guide to Cucumber acceptance tests for JS SDK
```
Comment (hjheath) [inline test/TESTING_GUIDELINES.md]: Make it clear that only Onfido can run these tests?

File: test/TESTING_GUIDELINES.md (hunk 1 above)
Comment (dawiddylowicz) [reply test/TESTING_GUIDELINES.md]: danke :)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-245-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://245-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-245-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://245-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://245-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by hjheath** [comment]
can you merge this?

**Pull request merged**